docker/images/n8n-custom/Dockerfile

```diff
@@ -20,7 +20,7 @@ COPY packages/nodes-base/ ./packages/nodes-base/
 COPY packages/workflow/ ./packages/workflow/
 RUN rm -rf node_modules packages/*/node_modules packages/*/dist
 
-RUN npm install --production --loglevel notice
+RUN npm ci --production --loglevel notice
```

Comment:
update docker build image to use `npm ci`. this will not update package lock and keep versions consistent with target versions when building the image.